src/plasmapy/analysis/swept_langmuir/plasma_potential.py

```diff
@@ -0,0 +1,249 @@
+"""
+Functionality for determining the plasma potential of a Langmuir sweep.
+"""
+
+__all__ = ["find_didv_peak"]
+__aliases__ = []
+__all__ += __aliases__
+
+import numbers
+from collections.abc import Sequence
+from typing import NamedTuple
+
+import numpy as np
+from scipy import signal
+
+from plasmapy.analysis.swept_langmuir.helpers import check_sweep, merge_voltage_clusters
+
+
+class dIdVExtras(NamedTuple):  # noqa: N801
```

Comment:
[optional] A possible alternative would be a `dataclass`.  I haven't thought about which would be better, though.

Reply:
see past conversation about dataclasses vs namedtuples...https://github.com/PlasmaPy/PlasmaPy/pull/889#discussion_r493451997

At this time I'm not inclined to make the change to dataclasses, since it would need to be changed for all other existing functionality to keep a consistent design pattern `(value, extras)`.